src/main/java/org/apache/maven/doxia/DefaultConverter.java

```diff
@@ -248,14 +249,21 @@ public void convert(InputFileWrapper input, OutputFileWrapper output)
                 for (File f : files) {
                     File relativeOutputDirectory = new File(
                             PathTool.getRelativeFilePath(input.getFile().getAbsolutePath(), f.getParent()));
-                    parse(f, input.getEncoding(), input.getFormat(), output, relativeOutputDirectory);
+                    convert(f, input.getEncoding(), input.getFormat(), output, relativeOutputDirectory);
+                    cleanUp(input, f);
                 }
             }
         } finally {
             stopPlexusContainer();
         }
     }
 
+    private void cleanUp(InputFileWrapper input, File f) {
+        if (input.cleanUp(f)) {
+            LOGGER.info("Removed input file \"{}\" after successfull conversion", f);
```

Comment:
successful

Reply:
Thanks, fixed in https://github.com/apache/maven-doxia-converter/pull/85/commits/338102ec326abbc8c0bc93f950ea6ba6309ba2a0.